data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/tracing/Tracing.java

```diff
@@ -57,46 +60,42 @@ public static void setup(final TracingConfig tracingConfig) {
       keyValue("loggingDebugEnabled", logger.isDebugEnabled())
     );
 
-    tracerManagement.updateActiveTraceConfig(
-      TraceConfig.getDefault()
-        .toBuilder()
-        .setSampler(Sampler.traceIdRatioBased(tracingConfig.getSamplingRate()))
-        .build()
+    SdkTracerProviderBuilder tracerProviderBuilder = SdkTracerProvider.builder();
+
+    tracerProviderBuilder.setResource(
+      Resource.create(Attributes.of(ResourceAttributes.SERVICE_NAME, SERVICE_NAME))
+    );
+    tracerProviderBuilder.setSampler(
+      Sampler.traceIdRatioBased(tracingConfig.getSamplingRate())
     );
 
     if (tracingConfig.getBackend().equals(Backend.ZIPKIN)) {
-
       logger.debug("Add Zipkin processor");
-
-      tracerManagement.addSpanProcessor(
+      tracerProviderBuilder.addSpanProcessor(
         BatchSpanProcessor
           .builder(zipkinExporter(tracingConfig))
-          .setExportOnlySampled(true)
```

Comment:
Can't be set anymore?

Reply:
It seems useful.

Reply:
Now you need to set it like this: https://github.com/knative-sandbox/eventing-kafka-broker/pull/587/files#diff-5251f57656ad46ff5e6738a59c628075ea3274249446aca9807bae90f77606b3R65

Reply:
That is the same as https://github.com/knative-sandbox/eventing-kafka-broker/pull/587/files#diff-5251f57656ad46ff5e6738a59c628075ea3274249446aca9807bae90f77606b3L63, this sets whether we export an unsampled trace or not based on that sampling ratio.

Reply:
From my understanding of the release notes, now this is just done by the ratio sampler

Reply:
Ok, thanks!
/lgtm